### ISSUE TITLE
Error when combining alias and use in an umbrella project

### ISSUE THREAD

**Issue by evnu** [msaraiva/elixir_sense#12]
`elixir_sense` (as of 747e8e4b69f7f83e6174639c31dcafe6a4091550) seems to stumble over a combination of `alias` and `use` in umbrella projects. For the example below, I am presented with this error message: `ast.ex:16: module Inner is not loaded and could not be found` when trying to get the location of `Inner` from within `use Inner` (found with `alchemist.vim` as of 76fac04 , see https://github.com/slashmili/alchemist.vim/issues/107#issuecomment-327104520).

```
mix new --umbrella project
cd project/apps
mix new a
mix new b
# 1) make :a an umbrella dep of :b in deps: {:a, in_umbrella: true}
# 2) then, paste some code
echo "defmodule A do\ndefmodule Inner do\n def __using__(_), do: :ok\nend\nend" > a/lib/a.ex
echo "defmodule B do\nalias A.Inner\nuse Inner\n\nend" > b/lib/b.ex
cd ..
mix compile # should compile properly
```

`elixir_sense` aborts parsing the rest of the module when this happens. Then, getting definition locations is not possible. Note that the example above works when using `use A.Inner`.

Is this enough information to reproduce the issue?

**Comment by evnu** [comment]
I can also reproduce this issue within a non-umbrella project:

```sh
cd /tmp
mix new non_umbrella && cd non_umbrella
printf "defmodule A do\nalias A.B\nuse B\nend\n" > lib/a.ex
mkdir lib/a
printf "defmodule A.B do\ndef __using__(_), do: :ok\nend\n" > lib/a/b.ex
```

Trying to retrieve the location of `B` within `use B` in `lib/a.ex` results in the error in `ast.ex`.

I tried to extract the request sent to elixir_sense from `alchemist.vim`'s [`ElixirSenseClient`](https://github.com/slashmili/alchemist.vim/blob/master/elixir_sense.py#L37) for this example. The request was:

```
{'auth_token': None,
 'request': 'definition',
 'payload': {'buffer':'defmodule A do\nalias A.B\nuse B\nend',
 'column': 5,
 'line': 3
}
```

**Comment by msaraiva** [comment]
This has been fixed on v1.0.1. Please let me known in case you still find any problem. Thanks for reporting this issue!

**Comment by evnu** [comment]
Thank you, I will give the new version a try and report back if I encounter this error again.
